src/mainwindow.cpp

```diff
@@ -1,12 +1,25 @@
 #include <QWebFrame>
+#include <QStandardPaths>
 
 #include "mainwindow.h"
 #include "view.h"
 
 
 MainWindow::MainWindow() : QMainWindow()
 {
+    QWebSettings::globalSettings()->setAttribute(QWebSettings::LocalStorageEnabled, true);
+
+    manager = new QNetworkAccessManager(this);
+    diskCache = new QNetworkDiskCache(this);
+    cacheLocation = QStandardPaths::writableLocation(QStandardPaths::CacheLocation);
+
+    diskCache->setCacheDirectory(cacheLocation);
+    manager->setCache(diskCache);
+
+
     view = new View(this);
+    view -> page()->setNetworkAccessManager(manager);
+    view -> page()->settings()->setMaximumPagesInCache(1);
```

Comment:
what's the point of setting this? Users don't interact with the screens and this affects history.